### PR TITLE
[Rule Tuning] [BUG] Revert PowerShell Query modifications from #2823

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

The query optimization PR (https://github.com/elastic/detection-rules/pull/2823) removed the quotes from some of the powershell KQL queries' keywords, introducing partial matches (FPs). Here is a summary of what this means from #2614

> Tune the rule to not have FPs due to partial match.
>
>Due to the nature of the `powershell.file.script_block_text` field (Some context can be found [here](https://github.com/elastic/integrations/pull/1931)), we need to double quote any field value that contains non-alphanumerical chars, because KQL will use them as delimiters and treat the text as separated keywords instead of a full text. For example:
>
> image

### CODE REVIEW

**Review by eric-forte-elastic — APPROVED** [review]
Manual review and discussion. 
LGTM :+1:

File: tests/test_all_rules.py [hunk 1]
```diff
@@ -59,7 +59,10 @@ def test_file_names(self):
     def test_all_rule_queries_optimized(self):
         """Ensure that every rule query is in optimized form."""
         for rule in self.production_rules:
-            if rule.contents.data.get("language") == "kuery":
+            if (
+                rule.contents.data.get("language") == "kuery" and "powershell.file.script_block_text"
```
Comment (terrancedejesus) [inline tests/test_all_rules.py:63]:
```suggestion
                rule.contents.data.get("language") == "kuery" and "powershell.file.script_block_text"
```

Can we add this to `definitions.py` as `QUERY_FIELD_OP_EXCEPTIONS = []`. Then import this and use here? More of a nit but just keeps code consistent.

Comment (w0rk3r) [reply tests/test_all_rules.py:63]: @eric-forte-elastic

Comment (eric-forte-elastic) [reply tests/test_all_rules.py]: Added to definitions and imported :+1:

**Review by terrancedejesus — APPROVED** [review]
Looks good. Glad this was caught. 👍🏽

**Review by Mikaayenson — APPROVED** [review]
LGTM

**Pull request merged**